### ISSUE TITLE
wsl2 was working flawlessly, but now throws "Error running browser: connect ECONNREFUSED 127.0.0.1..."

### ISSUE THREAD

**Issue by sammyp42** [microsoft/vscode-js-debug#961]
To test, I repeated exact same steps on Windows 10 and Windows 10 WSL2 (Ubuntu 20.04) -

1) Launch VSCode and open an empty folder. (From WSL2, run: `code ~/src/js` where `~/src/js` is an empty folder.
2) Add following default `launch` config to `.vscode\launch.json`

```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "pwa-chrome",
            "request": "launch",
            "name": "Launch Chrome against localhost",
            "url": "http://localhost:8080",
            "webRoot": "${workspaceFolder}"
        }
    ]
}
```

3) Press "F5" to start debugging.
4) In native Win 10, Chrome starts up and attempts to connect to the url defined in launch.json.  However, in WSL2, Chrome starts, but then stops on it's own and a few seconds later I see popup stating 'Unable to launch browser: "Timed out waiting for browser connection"' and a message on the bottom right stating "Error running browser: connect ECONNREFUSED 127.0.0.1:<port>" ... where <port> is a random port number.

This used to work properly a few days ago.  It seems a recent update on VSCode for WSL2 broke this, but I'm not 100% sure.  Attached is a trace file.
[vscode-debugadapter-e09e6357.json.gz](https://github.com/microsoft/vscode-js-debug/files/6256690/vscode-debugadapter-e09e6357.json.gz)

**Comment by sammyp42** [comment]
This is with Node v12.22.0 (haven't tried any other version).

**Comment by sammyp42** [comment]
After doing a little more digging, it appears that downgrading from 1.55.0 to 1.54.3 resolves the issue.  I have disabled auto update and will continue using 1.54.3 until this issue is resolved in production.  It appears the issue is actually fixed in the following commit https://github.com/microsoft/vscode-js-debug/commit/2994e0576d17a54e11132f92945039dd827e0d78 and I was able to confirm the fix using the "insider" build!  For now, will use 1.54.3 until 1.56 is released.  So I can track, can this issue be kept open until 1.56 is released?
